Added license requirement for schedule post feature in release notes (#7669)
mattermost/docs: source/about/mattermost-v10-changelog.md

#### User Interface (UI)
 - Pre-packaged Calls plugin [v1.3.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.3.0).
 - Downgraded Traditional Chinese language to Beta.

>  - Added a feature to schedule a post for the future. Scheduling a post requires a Professional/Enterprise license to activate.

Reviewer: ```suggestion
 - Added a feature to schedule a message at a future date (Professional and Enterprise plans).
```
